lib/ts/recipe/thirdpartyemailpassword/types.ts

```diff
@@ -315,3 +318,5 @@ export type APIInterface = {
         | undefined
         | ((input: { code: string; state: string; options: ThirdPartyAPIOptions; userContext: any }) => Promise<void>);
 };
+
+export type TypeThirdPartyEmailPasswordEmailDeliveryInput = TypeEmailPasswordEmailDeliveryInput;
```

Comment:
This is an incorrect type. The user object in this type != the user object in the thirdpartyemailpassword type.

Reply:
Not sure how to go forward solving this. Let's discuss over a call. Possible ways would be:
- have a common User Type which all recipes deal with
- re-declaring the `EmailDeliveryInput` in each Recipe causes an issue when the emailDelivery object is passed down to a child Recipe and the child recipe deals with emailDelivery of a specific `EmailDeliveryInput` only

Reply:
Ok. Let's discuss on a call.